v23.1/cdc-queries.md

```diff
@@ -1,17 +1,15 @@
 ---
-title: Change Data Capture Transformations
+title: Change Data Capture Queries
 summary: Filter the changefeed data that emits to the changefeed sink.
 toc: true
 docs_area: stream
 ---
```

Comment:
Question: should we add `key: cdc-queries.html` to match the v22.2 version? Also acceptable: Andrew doesn't understand what the `key` value is for, and can be ignored!

Reply:
So the `key` is mysterious to me... The intention is that it allows movement with the version dropdown when a page's name has been changed
I tried adding it to both and it wouldn't work... when I added it to one it worked. Having said that, I will double check when this goes live because I don't know if something could be different in preview vs. live